### PR TITLE
Add Inventory and Uniconfig 4.2.7

### PR DESCRIPTION
**Pull request by Jozefiel**

### CODE REVIEW

File: composefiles/swarm-uniflow.yml [hunk 1]
```diff
@@ -302,7 +302,43 @@ services:
...
+  inventory:
+    image: frinx/frinx-inventory-server:latest
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "3"
+        max-size: "10m"
+    environment:
+      - DATABASE_URL=postgres://postgres:postgres@uniconfig-postgres:5432/inventory?sslmode=disable
+      - UNICONFIG_API_URL=https://uniconfig:8181/rests
```
Comment (marosmars) [inline composefiles/swarm-uniflow.yml:315]: how does this work with multiple zones ?

also, isn't that value "uniconfig" in some env var ? and thus should be reused here ?

File: config/krakend/templates/inventory.tmpl [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                                "scope": ["request"],
+                                "name": "Content-Type",
+                                "value":  "application/json"
+                            }
+                        },
+                        {
+                            "header.Modifier": {
+                                "scope": ["request"],
+                                "name": "x-tenant-id",
+                                "value":  "frinx"
```
Comment (marosmars) [inline config/krakend/templates/inventory.tmpl:32]: the default multitenancy values are also used when accessing workflow-proxy and resource-manager right ? 

so we probably have them copy-pasted in a few places already, can we extract them into global env vars ?

**Review by marosmars — COMMENTED** [review]
In general, it is really not a good practice to combine 2 distinct changes into a single commit and a single PR.

Like in here, upgrading uniconfig and adding inventory.

So I would avoid doing that (almost) at all costs in the future.

File: composefiles/swarm-uniflow.yml (hunk 1 above)
Comment (Jozefiel) [reply composefiles/swarm-uniflow.yml:315]: Uniconfig env is not stored in the .env file. We are using ./generate_uc_compose.sh where service name (URL name) is explicitly generated. Also, the current version of the inventory server is not prepared for multi-zone deployment. Thus this URL is hardcoded just for single-node deployment at this moment.

File: config/krakend/templates/inventory.tmpl (hunk 2 above)
Comment (Jozefiel) [reply config/krakend/templates/inventory.tmpl:32]: KrakenD can't use env variables in nested configs. But common headers can be stored in 'partials' and then inserted into specific templates.

**Pull request merged**